app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -448,28 +445,34 @@ public void onTimerSet(long millis, boolean shakeToReset, boolean vibrate) {
                         PlaybackControlsDialog dialog = PlaybackControlsDialog.newInstance(isPlayingVideo);
                         dialog.show(getSupportFragmentManager(), "playback_controls");
                         break;
+                    case R.id.open_feed_item:
+                        if (feedItem != null) {
+                            Intent intent = MainActivity.getIntentToOpenFeed(this, feedItem.getFeedId());
+                            startActivity(intent);
+                        }
+                        break;
                     case R.id.visit_website_item:
                         Uri uri = Uri.parse(getWebsiteLinkWithFallback(media));
                         startActivity(new Intent(Intent.ACTION_VIEW, uri));
                         break;
                     case R.id.share_link_item:
                         if (media instanceof FeedMedia) {
```

Comment:
I think this should rather be `feedItem != null`. In this case, `media instanceof FeedMedia` implies that `feedItem != null` but I think it's more clear to write it explicitly.

Reply:
I assume you are referring to `static FeedItem getFeedItem()` helper: 
https://github.com/AntennaPod/AntennaPod/blob/e18d8f351476ede5047a08ec1f0dd3a9b45a8b99/app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java#L877

If not, please call out.

I initially had `feedIem != null` but (impulsively) followed Android Studio's advice to take it out.

Reply:
```
                        if (media instanceof FeedMedia) {
		                            ShareUtils.shareFeedItemDownloadLink(this, feedItem);
		                        }
```

I mean this `if` statement. The code guarded by that `if` does not use `media`, so it's either not necessary to check or the check should be changed.

Reply:
Got it. Thanks for pointing out.